h2o-r/h2o-package/R/kvstore.R

```diff
@@ -252,8 +252,9 @@ h2o.download_pojo <- function(model, path="", getjar=NULL, get_jar=TRUE) {
   pojoname = gsub("[+\\-* !@#$%^&()={}\\[\\]|;:'\"<>,.?/]","_",model_id,perl=T)
   
   file.path <- paste0(path, "/", pojoname, ".java")
-  if( path == "" ) cat(java)
-  else {
+  if( path == "" ){
```

Comment:
If it's not already, we should also have `h2o.download_mojo` path arg default to `NULL` as well.